test/dtypes/test_affine_quantized_tensor_parallel.py

```diff
@@ -1,12 +1,17 @@
 from torchao.testing.utils import copy_tests, TorchAOTensorParallelTestCase
 from torch.testing._internal.common_utils import run_tests
-from torchao.quantization import int8_weight_only
+from torchao.quantization import int8_weight_only, float8_weight_only
 
-class TestAffineQuantizedTensorParallel(TorchAOTensorParallelTestCase):
-    pass
+# class TestAffineQuantizedTensorParallel(TorchAOTensorParallelTestCase):
```

Comment:
please revert the changes for int8wo, we should make it explicit by spelling out the method here as well I think `QUANT_METHOD_FN = staticmethod(int8_weight_only)`